contrib/examples/walkthrough/ups-instance-default-sp.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: servicecatalog.k8s.io/v1alpha1
```

Comment:
@vaikas-google what do you think about moving the deployment of ups to be part of the svc-cat chart so its just deployed by default?  As @kibbles-n-bytes was saying, having a walkthru (which is meant to be a learning example) not show the use of plans might not be the best example. Of course we'd need a real broker to be deployed as part of the walkthru instead.